src/py42/_internal/clients/alerts.py

```diff
@@ -68,12 +82,17 @@ def _get_alert_rules(
         page_num=None,
         page_size=None,
     ):
+        # This method exists separately from `get_page()` because
+        # of the tenant ID parameter - trying to avoid it and avoid duplicate calls
```

Comment:
`get_current_tenant_id()` will only make a single request, then store the tenant_id on the class for future lookups. So I don't think the duplication of these methods is worth saving a few attribute lookups.

Reply:
I forgot that implementation detail, this is a good callout.